src/features/dataset/DatasetMiniHeader.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import classNames from 'classnames'
+
+import Button from '../../chrome/Button'
+import TextLink from '../../chrome/TextLink'
+import { Dataset, qriRefFromDataset } from '../../qri/dataset'
+import Icon from '../../chrome/Icon'
+import DownloadDatasetButton from '../download/DownloadDatasetButton'
+
+export interface DatasetHeaderProps {
+  dataset: Dataset
+  show: boolean
```

Comment:
Flip the logic for this, components should display by default. I'd change the name to `hide`